stringbasic/Source.cpp

```diff
@@ -0,0 +1,18 @@
+#include "String.h"
+
+
+int main() {
```

Comment:
Test your different CTORS in main :) 
(It's important to test all of your code!)